watcher/lib/influxdb/rollups.go

```diff
@@ -171,18 +133,48 @@ func validateParam(funcName, param string) error {
 	case constants.FunctionPercentile:
 		if value < 0 || value > 100 {
 			return trace.BadParameter(
-				"Percentile value must be between 0 and 100 (inclusive)")
+				"percentile value must be between 0 and 100 (inclusive)")
 		}
 	case constants.FunctionTop, constants.FunctionBottom, constants.FunctionSample:
 		if value < 0 {
 			return trace.BadParameter(
-				"Top, Bottom and Sample value must be greater or equal to 0")
+				"top, Bottom and Sample value must be greater than or equal to 0")
```

Comment:
Lowecase "bottom" and "sample" too, then?

Reply:
dumb me :) Fixed.